scripts/exregional_refl2tten.sh

```diff
@@ -227,6 +257,32 @@ bufr_table=${fixdir}/prepobs_prep_RAP.bufrtable
 # Fixed fields
 cp_vrfy $bufr_table prepobs_prep.bufrtable
 
+
+#-----------------------------------------------------------------------
+#
+# Build namelist and run executable 
+#
+#   fv3_io_layout_y : subdomain of restart files
+#
+#-----------------------------------------------------------------------
+
+if [ ${BKTYPE} -eq 1 ]; then
+  n_iolayouty=1
+else
+  n_iolayouty=$(($IO_LAYOUT_Y))
+fi
+
+cat << EOF > namelist_ref2tten
```

Comment:
do you want to make it consistent in the naming of namelist files for lightning, mosaic, and ref2tten? such as
namelist.lightning
namelist.mosaic
namelist.ref2tten